scripts/fix_lotus_encumbrances/fix_lotus_encumbrances.py

```diff
@@ -531,13 +621,15 @@ async def main():
 
     fiscal_year_id = get_fiscal_year_id(fiscal_year_code)
 
-    fund_ids = get_fy_fund_ids(fiscal_year_id)
+    all_funds = get_funds()
 
     # get open and closed order ids
-    closed_orders_ids = get_closed_orders_ids(fund_ids)
-    open_orders_ids = get_open_orders_ids(fund_ids)
+    closed_orders_ids = get_closed_orders_ids()
+    open_orders_ids = get_open_orders_ids()
     open_and_closed_orders_ids = closed_orders_ids + open_orders_ids
 
+    await fix_poline_encumbrances_relations(open_orders_ids, fiscal_year_id, all_funds)
```

Comment:
potentially (I can't say exactly) it's an issue to fix relations before we release encumbrances as some orders won't have links to unreleased encumbrances anymore